### ISSUE TITLE
CentOS /etc/hostname

### ISSUE THREAD

**Issue by lencap** [razorsedge/puppet-network#68]
CentOS 7 stores the hostname in /etc/hostname. Would updating global.pp with the following be a problem?

````
  if $hostname {
    file { 'network.hostname':
      ensure  => 'present',
      mode    => '0644',
      owner   => 'root',
      group   => 'root',
      path    => '/etc/hostname',
      content => "$hostname",
      notify  => Service['network'],
    }
  }
````

**Comment by razorsedge** [comment]
@lencap There would have to be a way to not write out that file on platforms that do not use it.  Support for EL7 has not progressed to the planning stage, but it is high time for me to get things working.  I just need to figure out when Fedora supported /etc/hostname.

**Comment by razorsedge** [comment]
Merged in efec40e35526b65240c0b8d0944a133dcb5e6d1f.
